src/utils/helpers/browser/browser.js

```diff
@@ -60,6 +60,16 @@ const Browser = {
   },
 
   /**
+   * Focues and sets cursor of input filed but does not select text
+   *
+   * @method setFocus
+   */
+  setFocus: (ref) => {
+    let node = ReactDOM.findDOMNode(ref._input);
```

Comment:
Can you add this line `// eslint-disable-line react/no-find-dom-node`

so that the final result is 
`let node = ReactDOM.findDOMNode(ref._input); // eslint-disable-line react/no-find-dom-node`

This should guard against future linting warnings that will come with carbon-factory.

Also is setFocus the correct name for this function. Would setInputFocus or something more specific as it requires a component to have a ref of `_input`